packages/icon/README.md

```diff
@@ -1,6 +1,8 @@
-# &lt;vaadin-icon&gt;
+# @vaadin/icon
 
-[&lt;vaadin-icon&gt;](https://vaadin.com/docs/latest/ds/components/icon) is a Web Component for creating SVG icons, part of the [Vaadin components](https://vaadin.com/docs/latest/ds/components).
+A web component for creating SVG icons.
```

Comment:
```suggestion
A web component for displaying SVG icons from an icon set.
```
Consider linking to vaadin-icon-set

Reply:
Thanks. Technically one could also provide standalone icon via `svg` property.

Reply:
Updated to use `displaying` instead.